catalyst_voices/packages/catalyst_voices_services/lib/src/user/user_service.dart

```diff
@@ -0,0 +1,223 @@
+import 'dart:async';
+
+import 'package:catalyst_cardano_serialization/catalyst_cardano_serialization.dart';
+import 'package:catalyst_voices_models/catalyst_voices_models.dart';
+import 'package:catalyst_voices_services/catalyst_voices_services.dart';
+import 'package:logging/logging.dart';
+
+abstract interface class UserService {
+  Account? get account;
+
+  Stream<Account?> get watchAccount;
+
+  Keychain? get keychain;
+
+  Future<List<Keychain>> get keychains;
+
+  Stream<Keychain?> get watchKeychain;
+
+  Future<void> useActiveAccount();
+
+  Future<void> removeActiveAccount();
+
+  Future<void> switchToAccount(Account account);
```

Comment:
selectAccount?

Reply:
im thinking about change all function to `useXYZ` so here `useAccount(Account account)`. What do you think?

Reply:
should be fine, lets do